cmake/FindMetaExamples.cmake

```diff
@@ -22,6 +22,8 @@ function(get_excluded_meta_examples)
     is_set(${SHOGUN_CONFIG} HAVE_LAPACK)
     is_set(${SHOGUN_CONFIG} USE_SVMLIGHT)
 
+  # TODO(geektoni): fix this meta example
+  LIST(APPEND evaluation/cross_validation_multiple_kernel_learning_weights_storage.sg)
```

Comment:
@geektoni I brought this file back to develop because otherwise the cookbook build fails. However the actual example doesn't seem to work (I pretty much copied what you did in #4655), so I black flagged it. It fails because the cross validation objects doesn't seems to have any observable values? I get ` Observation index (0) is out of bound (total observations 0)` in the .get_observation(0) line in the meta example.

Reply:
Mmmh I'll try to have a look. Maybe it is missing some observer...

Reply:
cool, thanks! this should be merged today into develop, so can fix it after that :)